contribute/documentation/README.md

```diff
@@ -1,7 +1,14 @@
 # Contribute to our documentation
 
-Welcome. We're glad you're here to help make our technical documentation even better.
+Welcome! We're glad you're here.
 
-For our style guide and writing guidelines, see [Writers' Toolkit](https://grafana.com/docs/writers-toolkit/).
+[Writers' Toolkit](https://grafana.com/docs/writers-toolkit/) is an open-source project intended to empower both internal and external contributors to help make our technical documentation even better.  
```

Comment:
We've inherited a rule from the original Vale Google style that says it should be "open source" not open-source but I've not got a citation for that beyond the rule itself: https://github.com/errata-ai/Google/blob/master/Google/WordList.yml#L66

Reply:
you're correct! I was using the grammatically correct, dictionary-approved spelling with a hyphen for the adjectival form, but Google says no hyphens.

Reply:
Thanks for checking.

I now tried searching for "google open source hyphen" and got https://opensource.google/documentation/reference/comms/style-guide#:~:text=Open%20source%3A%20Don't%20capitalize,(like%20Google%20Open%20Source) which does seem to be a citation I can use in future.

Reply:
Using open source.